### ISSUE TITLE
Consider A More Professional Sample Image

### ISSUE THREAD

**Issue by tjlanza** [CarlosFdez/world-explorer#5]
I don't know about anybody else, but your sample images were an immediate turn off.

**Comment by rsimoes** [comment]
I had to inform my players that we won't be doing hexploration because of the offensive image in the readme

**Comment by InfamousSky** [comment]
I agree, you missed out of some big time customers here, Supe. You won't sell many copies of your free module with such flagrant profanity on this christian website!

Until this is resolved I will take my business to other free modules!

**Comment by CarlosFdez** [comment]
I am considering replacing the image if I add enough features for this to be more than a toy with a niche, but it most likely wouldn't be to a professional image. Little things like this bring joy to what is normally thankless unpaid work that was developed to satisfy a personal need, shared because I see some other people also that same need.
